_online_courses/scala-zone.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: inner-page-no-masthead
+title: ScalaZONE Learning to Code in Scala
+edx-page: https://scala.zone
```

Comment:
The fact that it’s `edx-page` means that it will be shown with an edX icon like this one:
![Screenshot from 2020-11-25 16-37-59](https://user-images.githubusercontent.com/332812/100251120-b10f9b80-2f3e-11eb-9149-98884f036b21.png)
We should probably change the website to be able to show links that are not just edX or Coursera.